test/suite/test_tiered15.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_tiered15.py
+#   Test the "type" configuration in session.create with tiered storage.
+
+from helper_tiered import TieredConfigMixin, tiered_storage_sources
+from wtscenario import make_scenarios
+import wiredtiger, wttest
+
+class test_tiered15(TieredConfigMixin, wttest.WiredTigerTestCase):
+    types = [
+        ('file', dict(type='file', tiered_err=False, non_tiered_err=False, tiered_errmsg=None, non_tiered_errmsg=None)),
+        ('table', dict(type='table', tiered_err=True, non_tiered_err=False, tiered_errmsg="/Operation not supported/", non_tiered_errmsg=None)),
```

Comment:
Since this is all the same for all, I think `tiered_errmsg` can be pulled out and then just check for `self.type != 'file'` for the error message. The `non_tiered_errmsg` is varied enough to warrant keeping here.